wolfcrypt/src/asn.c

```diff
@@ -10653,7 +10653,7 @@ static const CertNameData certNameSubject[] = {
     /* Street Address */
     {
         "/street=", 8,
-#ifdef WOLFSSL_CERT_EXT
+#ifdef WOLFSSL_CERT_GEN
```

Comment:
Actually it's `#if defined(WOLFSSL_CERT_GEN) || defined(WOLFSSL_CERT_EXT)`. Please fix. Thanks

Reply:
Fixed.